### ISSUE TITLE
About the result

### ISSUE THREAD

**Issue by azraelkuan** [tuan3w/cnn_vocoder#3]
Hi, recently, i am also working on this paper, here is my [samples](https://soundcloud.com/dpm1b9vjiaap/sets/multi-head-cnn-inversion-spectrogram) 
The result shows that the high frequency can get the much the same as the origin, but the power in all the freq is much higher than the origin wav.

do u have any solutions for this question?

**Comment by hyzhan** [comment]
@azraelkuan What about the speed of the algorithm's audio generation? Is this generated with this repositorie?

**Comment by azraelkuan** [comment]
@hyzhan the speed is about 3000khz on 1080TI, i do not use this repo, i write by my own.

**Comment by hyzhan** [comment]
@azraelkuan After adding Spectral Convergence loss function of the paper seems to be able to better control the power of different frequency bands in this repo. Have you tried it?

**Comment by azraelkuan** [comment]
@hyzhan I have got a better result
[wavs.zip](https://github.com/tuan3w/cnn_vocoder/files/2564495/wavs.zip)

**Comment by hyzhan** [comment]
@azraelkuan It sounds great ! Is this generated by the original audio? I use this project to produce this result from the original audio. 
[generated_70k+.zip](https://github.com/tuan3w/cnn_vocoder/files/2564704/generated_70k%2B.zip)
 By using tacotron output to train the results, it can not hear the pronunciation of the audio.

**Comment by tuan3w** [comment]
@hyzhan Maybe you don't have enough data. Try to synthesis more data for training.
